path_oidc_test.go

```diff
@@ -365,6 +367,116 @@ func TestOIDC_AuthURL_namespace(t *testing.T) {
 	}
 }
 
+func TestOIDC_AuthURL_max_age(t *testing.T) {
+	b, storage := getBackend(t)
+
+	// Configure the backend
+	req := &logical.Request{
+		Operation: logical.UpdateOperation,
+		Path:      configPath,
+		Storage:   storage,
+		Data: map[string]interface{}{
+			"oidc_discovery_url": "https://team-vault.auth0.com/",
+			"oidc_client_id":     "abc",
+			"oidc_client_secret": "def",
+		},
+	}
+	resp, err := b.HandleRequest(context.Background(), req)
+	require.NoError(t, err)
+	require.False(t, resp.IsError())
+
+	tests := []struct {
+		name           string
```

Comment:
Since these test cases have a name, I'd prefer to see this collection as a map instead of a slice. It gives the name for free as the map key, and helps ensure there's no sharing of data between test runs since the tests are run in a different order each time.

Reply:
Will do 👍

Reply:
Done in 6928898.